SU-SOE/stanford_bean_types
# SOEOP-170: added opacity class to the featured image

# READY FOR REVIEW

# Summary
Add opacity classes so Tonja and friends can change the opacity on banner images.
_See ticket: https://stanfordits.atlassian.net/browse/SOEOP-170_ 

# Needed Review By 
12/11/2019

_For release: https://stanfordits.atlassian.net/browse/SOEOP-172_

# Criticality
- Fixed on prod, need to put into code.


# Steps to Test
- Checkout this branch
- Navigate to the homepage and click on "Configure Block" for the Banner image. 
- Add each of these classes (separately) `opacity-none`, `opacity-5`, `opacity-3`, and `opacity`
- Save and verify you see the opacity change on the banner image.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOEOP-170

## Related PRs

## More Information

## Folks to notify
@rmundstock 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
